KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml

```diff
@@ -538,10 +547,21 @@
       <p>This is the count of new desktop client_ids (profiles) that are created over the course of the year. A new Firefox profile counts to  this metric on the day we receive its first main ping.</p>
       <p>The 2021 target for cumulative new profiles is +5% over forecast. </p>
       </div>
-    row: 65
+    row: 67
     col: 0
-    width: 15
+    width: 24
     height: 9
+  - name: " (4)"
+    type: text
+    title_text: ''
+    subtitle_text: ''
+    body_text: "*Running slow? If no one has queried a slice before, it will take\
+      \ a 2-3 minutes to build the forecasting models. From then on, queries against\
+      \ that slice should run within 10-15 seconds.*"
```

Comment:
This is new text at the top of the dash.
<img width="1347" alt="Screen Shot 2021-03-26 at 2 41 00 PM" src="https://user-images.githubusercontent.com/20819040/112678430-4a918000-8e41-11eb-9e8e-7ba6d2040b4f.png">